public/js/lighthouse-chart.js

```diff
@@ -22,17 +22,41 @@ import Loader from './loader';
  */
 const CHART_BASE_URL = '/api/lighthouse/graph/';
 
-class LighthouseChart {
+export class LighthouseChart {
 
   constructor() {
     this.chartElement = document.getElementById('chart');
     this.loader = new Loader(this.chartElement, 'dark-primary-background');
   }
 
+  _loadChartsApi() {
+    if (window.google) {
+      console.log('Googler Charts Loader already loaded');      
+      return Promise.resolve();
+    }
+
+    console.log('Loading Googler Charts Loader');
+    return new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      script.defer = true;
+      script.src = 'https://www.gstatic.com/charts/loader.js';
+      script.onload = () => {
+        resolve();
+      }
+      script.onerror = () => {
+        reject();
+      }
+      document.querySelector('head').appendChild(script);
+    });
+  }
+
   load() {
     this.loader.show();
-    google.charts.load('current', {packages: ['annotationchart']});
-    google.charts.setOnLoadCallback(this.drawChart.bind(this));
+    this._loadChartsApi()
+      .then(() => {
```

Comment:
If `_loadChartsApi()` resolved to `google.charts` itself, we we could ignore the fact that that horrid global even exists:

```js
this._loadChartsAPI().then((charts) => { charts.load(...); });
```

(Could also rename it `getChartsAPI()`.)